pyproject.toml

```diff
@@ -92,3 +93,25 @@ module = [
   "traceloggingdynamic",
 ]
 ignore_missing_imports = true
+
+[tool.poe.tasks.install]
+shell = "for dir in . ni_measurementlink_generator/ examples/*/; do pushd $dir; poetry install; popd; done"
+
+[tool.poe.tasks.ni-python-styleguide]
+args = [
+  { name = "command", positional = true, required = true }
```

Comment:
does this allow flags? (like `fix --aggressive`)

Reply:
Not yet. Since this is a sequence task, I can add support for specific flags, but I don't think I can automatically pass extra arguments to the underlying tasks. It sounds like that is only supported for cmd and script tasks.

Reply:
I wasn't able to get this to work.

`{ name = "extra_args", positional = true, multiple = true }` doesn't accept options.

`{ name = "aggressive", options = ["--aggressive"], type = "boolean" }` sets `$aggressive` to `True`, which is not useful for passing the argument unchanged.

Reply:
I'm splitting the "ni-python-styleguide" task into "check-style" and "fix-style"